scripts/dispatch

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -ex
+
+REPO="https://api.github.com/repos/rancher/rancher/actions/workflows/replace-env-value.yml/dispatches"
+
+# Only use first two parts, delimited by dot
+# v2.6.5 becomes v2.6, end result: release/v2.6
+ACTION_TARGET_BRANCH="release/$(echo $DRONE_TAG | cut -f1,2 -d".")"
```

Comment:
`$DRONE_TAG` would include tags in other formats for the other products. I thought i saw an earlier version of this where they were excluded, but might have been wrong

Reply:
Please share the formats so it can be targeted for Rancher

Reply:
I think we decided to ensure tags for Rancher will follow usual semver (`v...`) whilst other products will prefix their name (`epinio-v...`, `desktop-v...`, etc).

Reply:
I added a check for starting with `v2`.